src/index.html

```diff
@@ -22,6 +22,377 @@
+              <span class="card__review-star"></span>
+            </div>
+            <span class="card__review-quantity">Reviews: 5</span>
+          </div>
+          <div class="card__price">
+            <span class="card__price-title">Price:</span>
+            <span class="card__price-cost">$2,199</span>
+          </div>
+
+          <div class="card__button">
+            <a
+              href="#"
+              class="card__link"
+            >
+              BUY
+            </a>
+          </div>
+        </div>
+
+        <div class="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="mac-book image"
+            class="card__image"
+          />
+          <div class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__product-code">Product code: 195434</div>
+          <div class="card__review">
+            <div class="card__review-stars card__review-stars--4">
+              <span class="card__review-star"></span>
+              <span class="card__review-star"></span>
+              <span class="card__review-star"></span>
+              <span class="card__review-star"></span>
+              <span class="card__review-star"></span>
+            </div>
+            <span class="card__review-quantity">Reviews: 5</span>
+          </div>
+          <div class="card__price">
+            <span class="card__price-title">Price:</span>
+            <span class="card__price-cost">$2,199</span>
+          </div>
+
+          <div class="card__button">
+            <a
+              href="#"
+              class="card__link"
+            >
+              BUY
+            </a>
+          </div>
+        </div>
+
+        <div class="card">
+          <img
+            src="./images/imac.jpeg"
+            alt="mac-book image"
+            class="card__image"
+          />
+          <div class="card__product-name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </div>
+          <div class="card__product-code">Product code: 195434</div>
+          <div class="card__review">
+            <div class="card__review-stars card__review-stars--4">
+              <span class="card__review-star"></span>
+              <span class="card__review-star"></span>
+              <span class="card__review-star"></span>
+              <span class="card__review-star"></span>
+              <span class="card__review-star"></span>
+            </div>
+            <span class="card__review-quantity">Reviews: 5</span>
+          </div>
+          <div class="card__price">
+            <span class="card__price-title">Price:</span>
+            <span class="card__price-cost">$2,199</span>
+          </div>
+
+          <div class="card__button">
+            <a
+              href="#"
+              class="card__link"
+            >
+              BUY
+            </a>
+          </div>
+        </div>
```

Comment:
The product card structure is repeated multiple times with the same content. Consider using a template engine or JavaScript to dynamically generate these cards if the content is meant to be the same or fetched from a data source.